projects/noloan2/app/src/main/java/hasadna/noloan2/LawsuitPdfActivity.java

```diff
@@ -0,0 +1,295 @@
+package hasadna.noloan2;
+
+import android.Manifest;
+import android.content.DialogInterface;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.os.Environment;
+import android.support.annotation.NonNull;
+import android.support.v4.app.ActivityCompat;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.itextpdf.licensekey.LicenseKey;
+import com.itextpdf.html2pdf.HtmlConverter;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+import noloan.R;
+
+public class LawsuitPdfActivity extends AppCompatActivity {
+  // TODO: Add logs to class functions.
+  private static final String TAG = "LawsuitPdfActivity";
+  private final int STORAGE_PERMISSION_CODE = 1;
+
+  // Folder names
+  private static final String LAWSUIT_MAIN_DIR_NAME = "lawsuits";
+  private static final String LAWSUIT_TEMPLATE_DIR_NAME = "templates";
+  private static final String LAWSUIT_OUTPUT_DIR_NAME = "pdf";
+  private static final String LAWSUIT_TEMPLATE_FILE_NAME = "template.xhtml";
+
+  // Folder paths - initDirStructure()
+  private String lawsuitMainPath = "";
+  private String lawsuitTemplateDirPath = "";
+  private String lawsuitTemplateFilePath = "";
+  private String lawsuitOutputPath = "";
+
+  // Date formats
+  private static final String TIME_ZONE = "Asia/Jerusalem";
+  private static final SimpleDateFormat DATE_TIME_FORMATTER =
+      new SimpleDateFormat("dd-M-yyyy hh-mm-ss");
+  private static final SimpleDateFormat DATE_FORMATTER = new SimpleDateFormat("dd-M-yyyy");
+
+  // Initialise until App has client-side form
```

Comment:
I would write this as a TODO:
`// TODO: Remove these once client-side form is done.

Reply:
Changed it.